api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
+
+/**
+ * Created by mjbarger on 9/6/16.
```

Comment:
Let's change the comment to talk about what the class represents and is used for.